Modules/Sources/DesignSystem/Foundation/BuildSettings+Brand.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+import BuildSettingsKit
+
+extension AppBrand {
+    static var current: AppBrand {
+        // TODO: remove this when unit tests not longer rely on `BuildSettings.current`.
```

Comment:
Unfortunately, I had to add this as there are too many unit tests that test UI components in the app.